### PR TITLE
October 18 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 3ccf541 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-547.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20241018_amplify.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+There are no known limitations in this release.
+
+### On-premise agents
+
+The helm based deployment of current and future On-prmise agents have been updated to use agent images from the [https://docker.repository.axway.com](https://docker.repository.axway.com). The use of a kubernetes secret is required.  This change has been made to the Axway API Management Gateway Agent, Istio, Graylog and Tracable on-premise agents.
+For more details, see:
+[Connect API Manager](https://content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md)
+[Connect Nesh Management](https://content/en/docs/connect_manage_environ/mesh_management/deploy-agents-with-helm.md)
+[Connect Graylog](https://content/en/docs/content/en/docs/runtime_security/configure_runtime_compliance_graylog.md)
+[Connect Traceable](https://content/en/docs/content/en/docs/runtime_security/configure_runtime_compliance_traceable.md)
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/20241018_amplify.md:72]: "https://content/en/docs" should be replaced with "/docs" for these links

Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20241018_amplify.md]: updated

**Review by vivekschauhan — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20241018_amplify.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+## Consumer experience
+
+### Marketplace
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **Filter pagination** - The filter on Marketplace have been enhanced to manage the pagination. Now, you will be able to scroll 3 times on a filters to see the current values. Then a **Load more** button appears to load the next set of data. Also, an indicator of the number of data for that filter is displayed on the top right corner of it. Refer to [Search and filter Marketplace content](/docs/manage_marketplace/consumer_experience/searching#filter-a-result)
+* **REason of Subscription cancellation** - The reason of subscription cancellation is now displayed in the subscription details so that consumer are aware why their subscription have been terminated. This cancellation can happen in 2 circumstances: when the subscription reached its term or when provider has activated the automatic subscription termination upon past due invoices.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20241018_amplify.md:180]: **Display of Subscription Cancellation reason** – The reason for a subscription cancellation is now shown in the subscription details, so users know why their subscription ended. There are two possible reasons for cancellation: the subscription has reached its end date, or the provider has automatically canceled it due to unpaid invoices.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/20241018_amplify.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+There are no known limitations in this release.
+
+### Service Registry
+
+There are no enhancements or bug fixes in this release.
+
+### Topology
+
+For customers which are unaware of new agent versions which have been released or have outdated agents, the **Agent Management** screen has been released within Topology.
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20241018_amplify.md]: edit made

File: content/en/docs/amplify_relnotes/20241018_amplify.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+There are no known limitations in this release.
+
+### SaaS (embedded) agents
+
+The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for SaaS (embedded) agents
+
+* **Azure Filter Settings** - The Azure SaaS agent has the ability to use a filter setting string.  This option can be used to only discover APIs which match the filter settings for an Azure SaaS agent.
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20241018_amplify.md]: added link to variables and examples.  @lbadenhop check the directory link please

**Pull request merged**